### PR TITLE
2.068 renames

### PR DESCRIPTION
**Pull request by CyberShadow**
Follow-up to the discussion:
http://forum.dlang.org/post/rvaguhviuldknnqktkbq@forum.dlang.org

~~For now this is a draft.~~ I'll be updating this up to RC1 following feedback.

@MartinNowak What's the ETA to RC1?

### CODE REVIEW

File: std/path.d [hunk 1]
```diff
@@ -1080,14 +1080,14 @@ auto setExt(R, C)(R path, C[] ext)
...
+    assert (withExtension("file"w, ".ext"w).array == "file.ext");
+    assert (withExtension("file.ext"w, ".").array == "file.");
 
     import std.utf : byChar, byWchar;
-    assert (setExt("file".byChar, "ext").array == "file.ext");
-    assert (setExt("file"w.byWchar, ".ext"w).array == "file.ext"w);
-    assert (setExt("file.ext"w.byWchar, ".").array == "file."w);
+    assert (withExtension("file".byChar, "ext").array == "file.ext");
+    assert (withExtension("file"w.byWchar, ".ext"w).array == "file.ext"w);
+    assert (withExtension("file.ext"w.byWchar, ".").array == "file."w);
```
Comment (CyberShadow) [inline std/path.d]: This example (documented unittest) would look much better if UFCS was used, should I update it?

**Comment by CyberShadow** [comment]
@MartinNowak ping

**Comment by CyberShadow** [comment]
@MartinNowak ping

**Comment by CyberShadow** [comment]
So, do I recreate this pull and target the stable branch?

**Comment by JinShil** [comment]
> So, do I recreate this pull and target the stable branch?

To increase its chances of getting pulled, I say "yes".

**Comment by DmitryOlshansky** [comment]
@CyberShadow yes, likely it needs to target both branches.

**Comment by CyberShadow** [comment]
https://github.com/D-Programming-Language/phobos/pull/3454

**Pull request closed without merging**